osu.Game/Online/Leaderboards/LeaderboardScore.cs

```diff
@@ -62,9 +68,10 @@ public LeaderboardScore(ScoreInfo score, int rank, bool allowHighlight = true)
         }
 
         [BackgroundDependencyLoader]
-        private void load(IAPIProvider api, OsuColour colour)
+        private void load(IAPIProvider api, OsuColour colour, DialogOverlay overlay)
```

Comment:
Use `[Resolved]` attribute instead:
```csharp
    [Resolved]
    private DialogOverlay dialogs { get; set; }
```

Reply:
Thanks for the suggestion. I have made the changes.